docs/reference/ros.md

```diff
@@ -28,46 +28,47 @@ To get our workspace configuration running on your computer:
-| Source the overlay | Run the `srcnew` terminal command |
-| Create a package with a node | Run the "new ament_(python\|cmake) package with a node" VS Code task |
+| Tutorial step                       | Sailbot Workspace configuration                                         |
+| ----------------------------------- | ----------------------------------------------------------------------- |
+| Install a package                   | All packages used in the tutorials are already installed (step 2 above) |
+| Clone a sample repo (ros_tutorials) | ros_tutorials is already cloned (step 3 above)                          |
+| Resolve dependencies                | Run the "install dependencies" VS Code task                             |
+| Build the workspace                 | Run the "Build" VS Code task, AKA ++ctrl+shift+b++                      |
+| Source the overlay                  | Run the `srcnew` terminal command                                       |
+| Create a package with a node        | Run the "new ament_(python\|cmake) package with a node" VS Code task    |
 
 ## Tutorials
 
-We encourage all software members to work through the tutorials listed below in order.
-For tutorials that have both C++ and Python versions, NET members should do the C++ version
-while CTRL and PATH members should do the Python version.
-
-[CLI Tools](https://docs.ros.org/en/eloquent/Tutorials.html#beginner-cli-tools){target=_blank}
-
-- Introducing turtlesim and rqt
-- Understanding ROS 2 nodes
-- Understanding ROS 2 topics
-- Understanding ROS 2 services
-- Understanding ROS 2 parameters
-- Understanding ROS 2 actions
-- Using rqt_console
-- Recording and playing back data
-
-[Client Libraries](https://docs.ros.org/en/eloquent/Tutorials.html#beginner-client-libraries){target=_blank}
-
-- Creating a workspace
-- Creating your first ROS 2 package
-- Writing a simple publisher and subscriber (C++ or Python)
-- Writing a simple service and client (C++ or Python)
-- Using parameters in a class (C++ or Python)
-- Getting started with ros2doctor
+We encourage all software members to work through the [ROS tutorials](https://docs.ros.org/en/humble/Tutorials.html){target=_blank}
+that are listed below in order. For tutorials that have both C++ and Python versions,
+NET members should do the C++ version while CTRL and PATH members should do the Python version.
+
+- Beginner: CLI tools
+    - Introducing `turtlesim` and `rqt`
+    - Understanding nodes
+    - Understanding topics
+    - Understanding services
+    - Understanding parameters
+    - Understanding actions
+    - Using `rqt_console` to view logs
+    - Recording and playing back data
+- Beginner: Client libraries
+    - Creating a workspace
+    - Creating a package
+    - Writing a simple publisher and subscriber (C++ or Python)
+    - Writing a simple service and client (C++ or Python)
+    - Using parameters in a class (C++ or Python)
+    - Using `ros2doctor` to identify issues
+- Demos
+    - Logging
```

Comment:
Adding content from the [intermediate section](https://docs.ros.org/en/humble/Tutorials/Intermediate.html) might be useful, including:
- Launch
- Testing